### PR TITLE
RK-006 Device Description

### PR DESCRIPTION
**Pull request by gritd**
Minor, but also a test to commit to device list

### CODE REVIEW

File: devices.ttl [hunk 1]
```diff
@@ -683,11 +683,20 @@ _:boss-rc-10r device:typeA true .
 _:boss-rc-10r device:uri "https://www.boss.info/global/products/rc-10r" .
 
 _:retrokits-RK-006 device:make "Retrokits" .
-_:retrokits-RK-006 device:model "RK-006" .
+_:retrokits-RK-006 device:model "RK-006 Hub" .
```
Comment (audionerd) [inline devices.ttl:686]: I prefer to have the `model` field be strictly the model number or identifier, and describe that model in the `notes` field. For example, a `notes` field like "Portable USB MIDI/Gate Hub" would be appropriate.

File: devices.ttl [hunk 2]
```diff
@@ -683,11 +683,20 @@ _:boss-rc-10r device:typeA true .
...
 _:retrokits-RK-006 device:uri "https://www.retrokits.com/shop/rk006" .
+_:retrokits-RK-006 device:details "TRS-A output tip switchable to Gate/PWM".
+
+_:retrokits-RK-008 device:make "Retrokits" .
+_:retrokits-RK-008 device:model "RK-008" .
+_:retrokits-RK-008 device:inputs 2 .
+_:retrokits-RK-008 device:outputs 2 .
+_:retrokits-RK-008 device:typeA true .
+_:retrokits-RK-008 device:uri "https://www.retrokits.com/shop/rk008" .
+_:retrokits-RK-008 device:details "New MIDI Device".
```
Comment (audionerd) [inline devices.ttl:699]: Has this device been officially released yet? Once the page referenced in `uri` is up, and if you could add more descriptive short-form `notes` and/or long-form `details`, I'd be happy to merge this.

**Comment by audionerd** [comment]
Thanks for the entry. I added a few code comments.

BTW It's a little easier for me to review and merge a PR for a single devices, vs a PR that has several changes across devices.

**Comment by gritd** [comment]
Hi Erik, thanks, I’ll look it through and see if I can adhere to it, must sound silly but we don’t really use GitHub for collab so I had to look op how to fork and merge something reliably first.

Meanwhile the RK008 is announced, it’s a ‘Real-time MIDI Recorder/Performer’ I can try my hand at editing again but you’re welcome to add it too of course :-)

Kind regards,
Gerrit

**Comment by audionerd** [comment]
OK, I just added the RK-006 and RK-008. :) Let me know if you have any changes, I can update.

**Pull request closed without merging**